pkg/handler/aws/emailContact/main.go

```diff
@@ -21,8 +23,8 @@ func handler(ctx context.Context, req events.APIGatewayV2HTTPRequest) (events.AP
 
```

Comment:
It is not part of this PR, but could you add a comment here: `// TODO implement POW verification using nonce (see https://github.com/CollActionteam/collaction_backend/issues/58)`

Reply:
okay